package.json

```diff
@@ -33,12 +34,14 @@
   },
   "homepage": "https://github.com/mironov/react-redux-loading-bar",
   "peerDependencies": {
+    "@types/react": ">=0.14.55",
```

Comment:
@larrydahooster I agree that this should not be added as peerDependency because types are not a requirement to make `react-redux-loading-bar` work. Pushed the v2.6.3 without that dependency.

@janslow please let me know if that works for you